avm/res/data-factory/factory/main.bicep

```diff
@@ -319,20 +325,20 @@ resource dataFactory_roleAssignments 'Microsoft.Authorization/roleAssignments@20
   }
 ]
 
-module dataFactory_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.7.1' = [
+module dataFactory_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.9.0' = [
   for (privateEndpoint, index) in (privateEndpoints ?? []): {
     name: '${uniqueString(deployment().name, location)}-dataFactory-PrivateEndpoint-${index}'
     scope: resourceGroup(privateEndpoint.?resourceGroupName ?? '')
     params: {
-      name: privateEndpoint.?name ?? 'pep-${last(split(dataFactory.id, '/'))}-${privateEndpoint.?service ?? 'dataFactory'}-${index}'
+      name: privateEndpoint.?name ?? 'pep-${last(split(dataFactory.id, '/'))}-${privateEndpoint.service}-${index}'
```

Comment:
is it not helpful that for an organisation that they have the pep string filled with some name that is valuable to distinguish what it is? Having the word dataFactory in here helps you to recognise quickly what this pep is for. And correct me if I am wrong but it shouldn't affect its operations, it just a name right?

Reply:
I am going to resolve this, as its naming, and we can add in if we need in another pr